bazelrunner/src/main/java/org/jetbrains/bsp/bazel/bazelrunner/BazelRunner.kt

```diff
@@ -15,6 +15,8 @@ class BazelRunner private constructor(
     companion object {
         private val LOGGER = LogManager.getLogger(BazelRunner::class.java)
 
+        private val CARGO_BAZEL_REPIN_ENV_VAR = "CARGO_BAZEL_REPIN" to "1"
```

Comment:
i'd decouple it from bazelrunner - we can add another method to `BazelRunnerBuilder` like `withProcessEnv` or similar and then use it in the code

Reply:
Done

Reply:
could you add an explanation what it does and why we need it?

Reply:
"Setting `CARGO_BAZEL_REPIN=1` updates `cargo_lockfile`(`Cargo.lock` file) based on dependencies specified in `manifest`(`Cargo.toml` file) and syncs `lockfile` (`Cargo.bazel.lock` file) with `cargo_lockfile`. Ensures that both Bazel and Cargo are using the same versions of dependencies. Mentioned `cargo_lockfile`, `lockfile` and `manifest` are defined in `crates_repository` from `rules_rust`, see: https://bazelbuild.github.io/rules_rust/crate_universe.html#crates_repository. In our server used only with `bazel build` command." - I added this comment in the place where this variable is set now